script_library/deployment-actions-kvm.sh

```diff
@@ -80,10 +80,8 @@ function clean_airship(){
         clean_action=" -e clean_action=$1"
         action_desc=$1
     fi
-    echo "DANGER ZONE. Set the env var 'DELETE_ANYWAY' to 'YES' to delete airship artifacts ( ${action_desc} ) in your userspace."
-    if [[ ${DELETE_ANYWAY:-"NO"} == "YES" ]]; then
-        run_ansible ${socok8s_absolute_dir}/playbooks/generic-clean_airship.yml ${clean_action}
-    fi
+    echo "DANGER ZONE. This delete airship artifacts ( ${action_desc} ) in your userspace."
```

Comment:
`This will delete all ariship artifacts...`

Reply:
should we also remove  the `DANGER ZONE` ?

Reply:
Do we still want to keep action_desc for dev options?

Reply:
> should we also remove the `DANGER ZONE` ?

Keeping it because we need that for internal dev environment. because that clean airship function is also called from other options.  Currently  user input / message only exist in remove_deployment. option.

Reply:
> Do we still want to keep action_desc for dev options?

Yeah we need action_dev for internal dev env .